src/styles/h5p-button.css

```diff
@@ -0,0 +1,283 @@
+/* NAVIGATION BUTTONS */
+.h5p-theme-nav-button,
+.h5p-theme .h5p-theme-nav-button {
+  background-color: var(--h5p-theme-secondary-cta-base);
+  color: var(--h5p-theme-secondary-contrast-cta);
+  font-size: var(--h5p-theme-font-size-m);
+  font-weight: bold;
+  border-radius: var(--h5p-theme-border-radius-medium);
+  display: inline-block;
+  padding: var(--h5p-theme-spacing-xs) var(--h5p-theme-spacing-s);
+  border: none;
+  transition: all 0.1s linear 0s;
+  cursor: pointer;
+}
+
+.h5p-theme-nav-button[aria-disabled="true"],
+.h5p-theme-nav-button:disabled {
+  opacity: 0.4;
+}
+
+.h5p-theme .h5p-theme-nav-button:hover {
+  background-color: var(--h5p-theme-secondary-cta-light);
+  color: var(--h5p-theme-secondary-contrast-cta);
+  border-color: var(--h5p-theme-secondary-cta-light);
+  ;
+}
+
+.h5p-theme .h5p-theme-nav-button:active {
+  background-color: var(--h5p-theme-secondary-cta-dark);
+  color: var(--h5p-theme-secondary-contrast-cta);
+  border-color: var(--h5p-theme-secondary-cta-dark);
+}
+
+.h5p-theme .h5p-theme-next {
+  justify-self: flex-end;
+  grid-column: 3;
+  min-width: max-content;
+}
+
+.h5p-theme-next:after {
+  font-family: 'h5p-theme';
+  content: "\e901";
+  margin-left: var(--h5p-theme-spacing-xxs);
+  display: inline-block;
+}
+
+.h5p-theme .h5p-theme-previous {
+  justify-self: flex-start;
+  min-width: max-content;
+}
+
+.h5p-theme-previous:before {
+  font-family: 'h5p-theme';
+  content: "\e900";
+  margin-right: var(--h5p-theme-spacing-xxs);
+  display: inline-block;
+}
+
+.h5p-theme-label {
+  display: inline-block;
+}
+
+/* If smaller than 700px */
+@container h5p-navigation (max-width: 700px) {
+  .h5p-theme-label {
+    display: none;
+  }
+
+  .h5p-theme-nav-button {
+    border-radius: 25rem;
+    height: var(--h5p-theme-spacing-xl);
+    width: var(--h5p-theme-spacing-xl);
+    padding: 0;
+  }
+
+  .h5p-theme-next:after,
+  .h5p-theme-previous:before {
+    margin: 0;
+  }
+
+  /* Primary CTA if inside navigation */
+  .h5p-navigation .h5p-theme-primary-cta {
+    height: var(--h5p-theme-spacing-xl);
+    width: var(--h5p-theme-spacing-xl);
+    padding: 0 !important;
+    text-indent: 0 !important;
+    /* TODO REMOVE IMPORTANT but make sure Finish button looks good in Flashcards and Question set */
+  }
+
+  .h5p-navigation .h5p-theme-primary-cta:hover {
+    height: var(--h5p-theme-spacing-xl);
+    width: var(--h5p-theme-spacing-xl);
+    padding: 0;
+  }
+
+  .h5p-navigation .h5p-theme-primary-cta span {
+    display: none;
+  }
+
+  .h5p-navigation .h5p-theme-primary-cta:before {
+    text-indent: 0;
+    position: unset;
+    width: 100%;
+    opacity: 1;
+    transform: translate(0, 0);
+  }
+
+  .h5p-navigation .h5p-theme-primary-cta:hover:before {
+    text-indent: 0;
+    transform: translate(0, 0);
+  }
+
+  .h5p-navigation .h5p-theme-secondary-cta:before {
+    margin-right: unset;
+  }
```

Comment:
The entire h5p-button.css file is just extracted from core, with the exception of these few lines. Needed to remove the margin right when we only have the icon.

Reply:
Ignore this.